js/views/components/Moderators.js

```diff
@@ -219,6 +218,8 @@ export default class extends baseVw {
                   if (eventData.error) {
                     // errors don't have a message id, check to see if the peerID matches
                     if (IDs.includes(eventData.peerId)) {
+                      // provide the expected capitalization of peerID
+                      eventData.peerID = eventData.peerId;
```

Comment:
I won't hold up merge for this, but it would have been nice to also add in a `delete eventData.peerId` in there.